virttest/utils_test/libvirt.py

```diff
@@ -3742,3 +3742,27 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+def clean_up_nwfilter_binding():
+    """
+    Clean up existed nwfilter binding
+
+    """
+    cmd_result = virsh.nwfilter_binding_list(debug=True)
+    binding_list = cmd_result.stdout_text.strip().splitlines()
+    binding_list = binding_list[2:]
+    result = []
+    # If binding list is not empty.
+    if binding_list:
+        for line in binding_list:
+            # Split on whitespace, assume 1 column
+            linesplit = line.split(None, 1)
+            result.append(linesplit[0])
+    logging.info("existed nwfilter binding list is: %s", result)
+    for binding_uuid in result:
+        try:
+            virsh.nwfilter_binding_delete(binding_uuid)
+        except Exception as e:
+            logging.error("Exception thrown while undefining nwfilter-binding: %s", str(e))
+            raise
```

Comment:
Could just use
```
for binding_uuid in result:
    virsh.nwfilter_binding_delete(binding_uuid, ignore_status=False)
```
meet your need? As it will call process.run and give clear cmd and result error log

Reply:
remove raise statement in order to allow it silently failed once happened.
The reason behind it is that it is just clean up functionality, we can not stop or fail case execution due to clean up failures.